rust/tools/authorization-logic/BUILD

```diff
@@ -118,6 +118,7 @@ rust_library(
         "//rust/tools/authorization-logic/cargo:antlr_rust",
         "//rust/tools/authorization-logic/cargo:bincode",
         "//rust/tools/authorization-logic/cargo:serde",
+        "//rust/tools/authorization-logic/cargo:structopt",
```

Comment:
You should probably make the diffbase of this PR the PR that introduces `structopt` so reviewers can focus on your new changes. It also adds a barrier to help prevent submitting the contents of the dependent PR before that PR.

Reply:
Looks like you fixed this.